librz/arch/isa/pic/pic_midrange_esil.inc

```diff
@@ -123,10 +123,8 @@ INST_HANDLER(BRA) {
 }
 
 INST_HANDLER(BRW) {
-	ut64 wreg;
 	op->type = RZ_ANALYSIS_OP_TYPE_UJMP;
-	rz_analysis_esil_reg_read(analysis->esil, "wreg", &wreg, NULL);
-	op->jump = addr + 2 * (wreg + 1);
+	op->jump = addr + 2; // not statically known: * (wreg + 1)
```

Comment:
If this is know known, shouldn't this jump be ut64_max?

Reply:
Yes, that's probably better. `addr + 2` does not really have much value.